### PR TITLE
Enhancement/meta info

### PR DESCRIPTION
**Pull request by DmytroNazarenko**
Add build meta info about contract

### CODE REVIEW

File: src/predeployed_generator/openzeppelin/proxy_admin_generator.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class ProxyAdminGenerator(OpenzeppelinContractGenerator):
     '''
 
     ARTIFACT_FILENAME = 'ProxyAdmin.json'
+    META_FILENAME = 'ProxyAdmin.meta.json'
```
Comment (DimaStebaev) [inline src/predeployed_generator/openzeppelin/proxy_admin_generator.py:19]: Is this file added to the package on build?

File: src/predeployed_generator/openzeppelin/transparent_upgradeable_proxy_generator.py [hunk 2]
```diff
@@ -17,6 +17,7 @@ class TransparentUpgradeableProxyGenerator(OpenzeppelinContractGenerator):
     '''Generates transparent upgradeable proxy
     '''
     ARTIFACT_FILENAME = 'TransparentUpgradeableProxy.json'
+    META_FILENAME = 'TransparentUpgradeableProxy.meta.json'
```
Comment (DimaStebaev) [inline src/predeployed_generator/openzeppelin/transparent_upgradeable_proxy_generator.py:20]: Is this file added to the package on build?

**Review by DimaStebaev — COMMENTED** [review]
I think changes worth increasing of the package minor version.

**Review by gannakulikova — DISMISSED** [review]

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.0.0
+1.0.1
```
Comment (DimaStebaev) [inline VERSION]: A new functionality was added.
`1.1.0`

File: scripts/generate_meta.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import json
+import os
+from os.path import normpath, join, dirname, basename
+
+dbg_path = os.getenv('DBG_PATH')
+package_artifacts_path = os.getenv('ARTIFACTS_DIR')
+
+
+def get_build_info_path():
```
Comment (DimaStebaev) [inline scripts/generate_meta.py:9]: What do you think about specifying of return value type?

**Review by DimaStebaev — APPROVED** [review]

**Pull request merged**